www/src/components/profile/MyProfile.tsx

```diff
@@ -26,18 +26,24 @@ const DIRECTORY = [
 ]
 
 export function MyProfile() {
-  const { me } = useContext(CurrentUserContext)
+  const me = useContext(CurrentUserContext)
   const { pathname } = useLocation()
   const tabStateRef = useRef<any>(null)
   const currentTab = DIRECTORY.find(tab => pathname?.startsWith(tab.path))
 
+  let url: string | undefined
+
+  if (me.avatar != null) {
+    url = me.avatar
+  }
+
```

Comment:
```suggestion
  let url = me.avatar || undefined
```
this will catch the empty string as well as being terser